0.Exploratory_Data_Analysis/nbconverted/Exploratory_Data_Analysis.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Exploratory Data Analysis on Wave Data
+
+# ### Wave1 dilated 50 data
+# Here we understand more about the data and how it is distrubuted.
+# Wave1 data are extracted features from raw images.
+# These images were processed via Cellprofiler pipelines
+#
+# Specifically wave1 is looking at Gasdermin-D and Nuclei Staining from a cell painting experiment.
+#
+# Further, nuclei were dilated using multiple values of pixel dilation. Here we use data for the 50 pixel dialation
+
+#
+# ### Wave3
+# Here we look into how the data are distributed in wave3, the same plate data as wave1 but with full cell painting dataset including the Gasdermin D channel for a total of 6 channels to analyze.
+# These data sets contain extracted features from raw image dataset. See README for information as to how raw image data was processed.
+
+# In[1]:
+
+
+import sys
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+get_ipython().run_line_magic("matplotlib", "inline")
+import itertools
+
+from sklearn.cluster import KMeans
+from sklearn.manifold import TSNE
+
+sys.path.append("..")
+from ..utils.utils import *
+
+# In[2]:
+
+
+# function for EDA of each input feature extraction file:
+def EDA_run(df, wave_label):
+    """Function runs EDA on imported data frame
+
+    Parameters:
+    df: pandas dataframe
+    wave_label: string of label for outputed graphs
+
+    Return:
+    graphs, and df stats
+
+    """
+    # Call function to display df shape and # of replicates present
+    df_stats(df)
```

Comment:
Does `df_stats` come from the `utils` module?

Reply:
yes